dev/tests/integration/testsuite/Magento/Store/Model/GetDefaultStoreGroupTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Store\Model;
+
+/**
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
+ * phpcs:disable Magento2.Security.Superglobal
+ */
+class GetDefaultStoreGroupTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * @var \Magento\Store\Model\GetDefaultStoreGroup
+     */
+    protected $getDefaultStoreGroupModel;
+
+    /**
+     * @inheritdoc
+     */
+    protected function setUp(): void
+    {
+        $this->getDefaultStoreGroupModel = \Magento\TestFramework\Helper\Bootstrap::getObjectManager()->get(
+            \Magento\Store\Model\GetDefaultStoreGroup::class
+        );
+    }
+
+    /**
+     * @inheritdoc
+     */
+    protected function tearDown(): void
```

Comment:
I'd remove setup and teardown methods and use $getDefaultStoreGroupModel as a variable in the test method for simplification